pkg/helpers/Get-AzureConstants.py

```diff
@@ -61,8 +49,10 @@ def getStorageAccountType(sizeName):
     else:
         return "Standard_LRS"
 
-def getFileContents(masterAgentMap, kubernetesAgentMap, sizeMap, locations):
-    text = r"""package helpers
+def getFileContents(kubernetesSizeMap, locations):
+    text = r"""// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
why is there a copyright comment here?

Reply:
We need it to add it here. SInce `azure_const.go` is an AUTOGENERATED file, the edits have to be done through here